src/rleCompress.js

```diff
@@ -18,7 +18,36 @@
  * @return {string}
  */
 function rleCompress(source) {
-  // write code here
+  let sourceCompress = '';
```

Comment:
название переменной в этом случае должно называться как результат какого-то действия, а не быть похожим на глагол
```suggestion
  let compressedSource = '';
```

Reply:
Got it